decidim-core/app/views/pages/terms.ca.html.erb

```diff
@@ -0,0 +1,4 @@
+<center>
+  <h1>Substitueix aquest text pels termes i condicions.</h1>
+  <p>Pots fer-ho creant un arxiu a app/pages/terms.ca.html.erb</p>
```

Comment:
I'd wrap the filename in `<pre>` keys or something similar (I think `code` also exists)